src/devices/is42s32400f.rs

```diff
@@ -0,0 +1,57 @@
+/// ISI IS42S32400F SDRAM
+#[allow(unused)]
+
+/// Speed Grade 6
+pub mod is42s32800g_6 {
```

Comment:
I think the mod name should be changed

Reply:
Woops, forgot to change this while copying, thanks!